src/Plugins/OrientationAnalysis/src/OrientationAnalysis/Filters/Algorithms/GenerateGBCDPoleFigure.cpp

```diff
@@ -326,7 +326,7 @@ Result<> GenerateGBCDPoleFigure::operator()()
 
   ParallelData2DAlgorithm dataAlg;
   dataAlg.setRange(0, xPoints, 0, yPoints);
-  dataAlg.setParallelizationEnabled(true);
+  dataAlg.setParallelizationEnabled(false);
```

Comment:
Check if parallelization disabled is intended.

Reply:
@mmarineBlueQuartz Can we check if the DataArray is out-of-core then turn off parallel otherwise leave it on?